### DISCUSSION TITLE
[THEME] Mist-Moon for Trilium

### DISCUSSION THREAD

**Discussion by Ivy-End** [TriliumNext/Trilium#4218] [Show and tell]
It's a theme that suits Chinese typing habits and integrates Obisidian's bright color scheme.

[https://github.com/Ivy-End/Mist-Moon](https://github.com/Ivy-End/Mist-Moon)

![image](https://github.com/zadam/trilium/assets/3644544/8eab154e-1f50-436c-94e4-ef6195a17d7a)

**Comment by Nyar233** [comment]
It's really pretty.

**Reply by Ivy-End** [reply]
It's great to have your compliments!
